src/api/app/models/workflow_run.rb

```diff
@@ -0,0 +1,3 @@
+class WorkflowRun < ApplicationRecord
```

Comment:
This model should have validations.

Reply:
Model validations added. The migration was also adapted. Please have a look again.

Reply:
I would add a few more validations:

- `status` has to be one of the valid values with [inclusion](https://guides.rubyonrails.org/active_record_validations.html#inclusion).
- Length of the various string/text columns

Reply:
Regarding the `status` we don't need to add an extra validation. It's an enum and enum will only allow the values defined in the model. 
And we are not sure about the length of the string/text columns. Because it's a payload and it's huge. I'd leave it like that. What do you think?

Reply:
Oh right, there's a built-in validation with the enum, then it's all good :+1:.

I think validating the length of `response_url` should be enough.

Reply:
Validation for the length of `response_url` added. Please have a look.